modules/layers/src/layers/editable-geojson-layer.ts

```diff
@@ -299,15 +306,10 @@ export default class EditableGeoJsonLayer extends EditableLayer {
           mode = props.mode;
```

Comment:
if `mode` isnt `string` or `function`, what it would be?

Reply:
An instance of an EditMode.